src/test/java/part1/exercise/StreamsExercise1.java

```diff
@@ -49,11 +90,12 @@ public void sumEpamDurations() {
             }
         }
 
-        // TODO
-        throw new UnsupportedOperationException();
+        int result = employees.stream()
+                .flatMap(e -> e.getJobHistory().stream().filter(j -> j.getEmployer().equals("epam")))

```

Comment:
You could simplify this: `.flatMap(e -> e.getJobHistory().stream()).filter(...)`